app/wp-content/themes/kstg/library/js/vehicle.js

```diff
@@ -7,6 +7,18 @@ var calloutValue = $('.specialFeature').text();
 
   $(document).ready(function() {
 
+    //function for changing hero image
+
+    //function for highlighting selected image in carousel
+    function setActiveImage(event) {
+      var currentImage = $('ul').find('.borderSelector').toggleClass('borderSelector');
+      console.log(currentImage);
+      var $getImage = (event.target.id);
```

Comment:
It doesn't look like you're ever using this variable. 

FYI, the name is a little funny. I would name it something like `image` instead. If you were storing a `function` that gets an image, then the `getImage` name would make sense.